tests/lighthouse/tests/suites/assert/account_data.rs

```diff
@@ -22,12 +22,13 @@ async fn simple() {
         .unwrap();
 
     let test_account = create_test_account(context, &user, false).await.unwrap();
+    let log_level = lighthouse_sdk::types::LogLevel::Silent;
 
     let mut tx = TxBuilder {
         ixs: vec![
             AssertAccountDataBuilder::new()
                 .target_account(test_account.encodable_pubkey())
-                .log_level(lighthouse_sdk::types::LogLevel::PlaintextMessage)
+                .log_level(log_level.clone())
```

Comment:
Can you implement `Copy` on `LogLevel` so you don't need to invoke clone? It's an enum so the copy should be free anyways

Reply:
I've asked the kinobi contributes if they can add it to the codegen